packages/itwinui-css/src/alert/alert.scss

```
@@ -72,7 +72,7 @@
     @include iui-anchor-status($statusColor);
   }
 
-  #{$rootSelector}-button {
+  #{$rootSelector}-button:focus-visible {
```

`:focus` will be better than `:focus-visible` because it will work with the fallback for older browsers. (The fallback code is already set inside `iui-alert-button` through `@include iui-button`).

```suggestion
  #{$rootSelector}-button:focus {
```

For more context: this works because `:focus-visible` always implies `:focus` (but not vice versa). And it doesn't have an adverse effect because setting only the color on `:focus` has no effect unless the whole `outline` is also set.

---

That makes sense. I have updated and tested the change and confirm that it still behaves as expected. Thank you.